source/changelog.txt

```diff
@@ -17,6 +17,20 @@ v2.0.0
 
 *Upcoming*
 
+- Added support for these constructors:
+
+  - ``Binary.createFromBase64()``
+  - ``ObjectId.createFromHexString()``
+
+- ``mongosh`` returns binary values as ``Binary.createFromBase64(
+  <base64String> )`` values instead of ``Binary( Buffer.from(
+  <base64String> ) )`` values. For example:
+
+  - Starting in 2.0: ``binaryValue:
+    Binary.createFromBase64("SGVsbG8gV29ybGQhCg==")``
+  - Earlier versions: ``binaryValue:
+    Binary(Buffer.from("SGVsbG8gV29ybGQhCg==", "hex"), 0)``
+
```

Comment:
(linking)
Consider adding a link to where folks can get the additional information. I know this probably requires a follow on update, once things are linkable across repos but in this case I think it's worth the effort.